DH_Interface/Classes/DHDeployMenu.uc

```diff
@@ -1901,7 +1901,7 @@ defaultproperties
     VehicleUnavailableString="The vehicle you had selected is no longer available."
     LockText="Lock"
     UnlockText="Unlock"
-    CantLock="Squad can be locked only if it has more than 3 members"
+    CantLock="Squad can be locked only if it has 3 or more members"
```

Comment:
You can inject variables into strings like this:
`"Squad can be locked only if it has {0} or more members"`

And then swap `{0}` with the actual variable: 
`Repl(default.CantLock, "{0}", SRI.SquadLockMemberCountMin)`